project/settings.py

```diff
@@ -317,13 +317,20 @@
         'task': 'bots.elasticsearch.tasks.elasticsearch_janitor',
         'schedule': crontab(hour=23, minute=30),
     },
+}
 
-    # Once an hour
-    'Schedule Harvests': {
-        'task': 'share.tasks.schedule_harvests',
-        'schedule': crontab(minute=0)
+if not DEBUG:
+    CELERY_BEAT_SCHEDULE = {
+        **CELERY_BEAT_SCHEDULE,
+        'Schedule Harvests': {
+            'task': 'share.tasks.schedule_harvests',
+            'schedule': crontab(minute=0)
+        },
+        'RawData Janitor': {
+            'task': 'share.janitor.tasks.rawdata_janitor',
+            'schedule': crontab(hour=0)  # hourly
```

Comment:
Wouldn't hourly be `crontab(minute=0)`? Since `minute` defaults to `*`, I think this'll run every minute from midnight to 12:59.